Decrease memory-usage of subscriber import

jamesricky opened the pull request:
Decrease memory usage by clearing cached rows after every 100 subscribers imported.

This should be sufficient for a php memory_limit of 128M.

jamesricky committed:
Increase memory-limit for subscriber import

Prevents out-of-memory errors when importing multiple thousand subscribers at once.

kaufmo commented on Bundle/Command/ImportSubscribers.php:
```diff
@@ -136,6 +136,7 @@ protected function interact(InputInterface $input, OutputInterface $output)
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        ini_set('memory_limit', -1);
```
I think -1 is not good, eventually add a option to override it if necessary but not with -1

jamesricky committed:
Decrease memory-usage of subscriber import

Decrease memory usage by clearing cached rows after every 100
subscribers imported.

This should be sufficient for a php memory_limit of 128M.
